src/widgets/mainwindow.cpp

```diff
@@ -595,6 +596,19 @@ void MainWindow::closeOnQuit()
 
 void MainWindow::setupShortcuts()
 {
+    const auto &coreConfig = ConfigMgr::getInst().getCoreConfig();
+
+    // For cross-platform global shortcuts, the external library QHotkey is used
+    auto qHotkey = new QHotkey(QKeySequence(coreConfig.getShortcut(CoreConfig::WakeUp)), true, qApp);
+
+    // WakeUp
+    {
+        if (qHotkey) {
```

Comment:
It will always be true? Check the code of QHotkey to use the right way on judging invalid shortcut.

Reply:
done

Reply:
QHotkey after registration, it will automatically help us determine whether the shortcut key is effective, so it has been omitted here.